### ISSUE TITLE
Create a wiki-like interface for Congressional Districts

### ISSUE THREAD

**Issue by saikat** [BrandNewCongress/welcome#23]
Right now, we have quantitative data for a lot of Congressional Districts in Airtable + qualitative data on our Wiki.  What we'd like to do is merge all this data into Airtable, but then create a wiki-like interface to read, search, and edit this data.  So this interface will:

* Be able to interpret markdown in long text fields
* Have a main view that is the qualitative info about the district + also have quantitative info about the district somewhere
* Have an interface to allow people to edit information. To start, to make things simple, we should make it a system that doesn't require any logins and submits the edits as requests for editing
* Have a way for people to tell us about incorrect information.  To start, this can just be a link to our helpdesk e-mail :)
* The ability to add in and view attachments for research on incumbent pages, etc.

I'm imagining that there will be two kinds of pages here:
1) Congressional District or Senate page. This will have information on the district/senate seat
2) Incumbent page. This will have information on the current incumbent

It has a lot of similarities to ballotpedia, though a lot of what we are tracking is different and the relative importance of some information is different as well.  Also, we are collecting much more qualitative data through phone interviews/emails/one-on-one chats than ballotpedia has.

I'm currently looking for a designer to help me mock this up and then a developer or two to help put it together!

**Comment by ben-pr-p** [comment]
You could host a fork of mediawiki or docuwiki with a custom style layer? Those are both pretty customizable (an account is required to login, an account isn't, etc)

**Comment by saikat** [comment]
Yeah, thought about that.  The issue is mainly that mediawiki's UI (not just the aesthetic parts, the actual UX of it) is pretty awful and none of our volunteers have figured out how to make edits to a page without us first telling them it's an option.  So I want to think of a design a bit more from scratch, to start.  And then, if that design is easy to implement by forking mediawiki -- let's go that route.  But my guess is that the amount of work to make mediawiki conform to an intuitive design is going to be more than the amount of work of just building it from scratch.  

Basically, I want to think of this project from a design perspective first so we can be sure to end up with something that is usable for both a) people needing to access the info and b) people needing to update the info.

**Comment by ben-pr-p** [comment]
Oh hm, I think it's hard for me to see because I have made lots of MediaWiki edits in the past. I suppose if you go mock up first, devs can decide which is the closest to fork and then do that.

Either way, a reliable permissions + version control system is a bit of an undertaking I think for such a specific use case, and there are wiki clients written in pretty much every language I believe. 

For example, this Node based one https://github.com/claudioc/jingo renders Github flavored Markdown and interacting with its codebase seems less daunting than the gigantic Mediawiki

**Comment by saikat** [comment]
We probably also want to list local resources (like local groups + contacts and local press) on these pages.
